app/src/main/java/com/portfolio/tasky/usecases/PasswordPatternValidatorImpl.kt

```diff
@@ -0,0 +1,9 @@
+package com.portfolio.tasky.usecases
+
+import javax.inject.Inject
+
+class PasswordPatternValidatorImpl @Inject constructor(): PasswordPatternValidator {
```

Comment:
Same as "NamePatternValidatorImpl" comment

+ here you would check for:
At least 1 lower case letter
At least 1 upper case letter
At least 1 number
Minimum length 8 (or 9, can't really remember)

Reply:
Yes , this is what the regex has been configured for. 1 number, 1 upper-case, 1 lowe-case and minimum length 9.